randovania/games/samus_returns/logic_database/Area 3 - Factory Exterior.txt

```diff
@@ -818,15 +823,16 @@ Extra - asset_id: collision_camera_038
   > Event - Gamma Metroid
       All of the following:
           Defeat Gamma Metroid
-          # Destroy Bomb Block
-          Lay Any Bomb
-          Any of the following:
-              # Get Up
-              Single-wall Wall Jump (Intermediate) or Climb Rooms Vertically (High Jump)
-              # WJ: https://youtu.be/-QRvPeCbaOM
-              Wall Jump (Intermediate)
-              # SuperJump+UE
-              Super Jump (Intermediate) and Unmorph Extend (Beginner)
+          All of the following:
+              # Destroy Bomb Block
```

Comment:
No combining with the Gamma template? Would read `Defeat Gamma and Lay Any Bomb`, with a comment that says `Break the Bomb blocks to reach the Gamma`.  I think it's be cleaner at least, buy I'll leave that up to you

Reply:
I feel like this is nicer.

Reply:
Honestly I think a comment here is unnecessary but whatever